modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/CheckboxTag.java

```diff
@@ -41,10 +41,26 @@ public boolean getChecked() {
 		return _checked;
 	}
 
+	public String getCssClass() {
+		return _cssClass;
+	}
+
 	public boolean getDisabled() {
 		return _disabled;
 	}
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link #getCssClass()}
+	 */
+	@Deprecated
+	public String getElementClasses() {
```

Comment:
You  don't need to add `getElementClasses` nor `seElementClasses` since they're already defined in `BaseContainer` and this class extends from it.